src/actors/workers/database.rs

```diff
@@ -96,7 +96,7 @@ impl<B: BlockT> DatabaseActor<B> {
     async fn batch_storage_handler(&self, storage: Vec<Storage<B>>) -> Result<()> {
         let mut conn = self.db.conn().await?;
         let mut block_nums: Vec<u32> = storage.iter().map(|s| s.block_num()).collect();
-        block_nums.sort();
+        block_nums.sort_unstable();
```

Comment:
What did clippy complain about here?

Reply:
I think it's this lint: https://github.com/rust-lang/rust-clippy/issues/5762

Reply:
I see, TIL. :)